src/app/ui/components/virtuoso.tsx

```diff
@@ -20,7 +20,7 @@ export function VirtuosoWrapper({ children, hasFooter, isPopup }: VirtuosoWrappe
   const [key, setKey] = useState(0);
   const isAtLeastMd = useViewportMinWidth('md');
   const virtualHeight = isAtLeastMd ? '70vh' : '100vh';
-  const headerHeight = isPopup ? 230 : 60;
+  const headerHeight = isPopup ? 300 : 80;
```

Comment:
Seems like a bad choice to me that the core virtuoso component has these hard coded header/footer heights.

What if i use the component again with a different footer?

Reply:
This code isn't ideal but we have had this issue for a long time and this was the simplest fix. We only use this 3 times and twice with a footer. 

We could refactor to pass the values in but I think it's OK